emails/models.py

```diff
@@ -54,7 +54,9 @@ def valid_available_subdomain(subdomain, *args, **kwargs):
     #   can't have "blocked" words in them
     blocked_word = is_blocklisted(subdomain)
     #   can't be taken by someone else
-    taken = Profile.objects.filter(subdomain=subdomain).count() > 0
+    taken = RegisteredSubdomain.objects.filter(
+        subdomain_hash=hash_subdomain(subdomain)
+    ).count() > 0
```

Comment:
We need to make this either/or, since we have existing subdomains that are only in `Profile`.

```suggestion
    registered_before = RegisteredSubdomain.objects.filter(
        subdomain_hash=hash_subdomain(subdomain)
    ).count() > 0
    profile_currently_has = Profile.objects.filter(
        subdomain=subdomain
    ).count() > 0
    taken = registered_before or profile_currently_has
```

Reply:
Done with migration `emails/migrations/0028_copy_subdomain_to_registeredsubdomain.py`